src/clr-angular/forms/common/label.ts

```diff
@@ -29,6 +31,10 @@ export class ClrLabel implements OnInit, OnDestroy {
     if (this.ifErrorService) {
       this.renderer.addClass(this.el.nativeElement, 'clr-control-label');
     }
+    if (this.ifErrorService && !this.layoutService.isVertical()) {
+      this.renderer.addClass(this.el.nativeElement, 'clr-col-xs-12');
+      this.renderer.addClass(this.el.nativeElement, 'clr-col-md-2');
```

Comment:
Why `clr-col-md-2` as the default? What happens if the user requires the label to occupy more width? 

For eg: What happens if someone uses a vertical form in a small modal and the breakpoint is greater than `md`?

Reply:
This is the default based on what aligns with the design best, but it does not have an option to override or change it.

I could have it first check if `clr-col-xxx` classes exist and only apply these if they are not present.